connectors/src/connectors/snowflake/temporal/client.ts

```diff
@@ -40,11 +40,13 @@ export async function launchSnowflakeSyncWorkflow(
         connectorId: [connectorId],
       },
       signal: resyncSignal,
+      // If we don't pass signalArgs the workflow will not be signaled.
+      signalArgs: [],
       memo: {
         connectorId,
       },
       // Every 6 hours.
-      cronSchedule: "0 0 */6 * * *",
+      cronSchedule: "0 */6 * * *",
```

Comment:
As someone who waited for metabase sync and considering the super fast workflow, I would put it much more frequently.